### ISSUE TITLE
HydraHD.me / .cc / .co

### ISSUE THREAD

**Issue by spoon-codes** [PreMiD/Presences#9028]
### Website name

HydraHD

### Website URL

hydrahd.me, hydrahd.cc, hydrahd.com

### Website logo

https://hydrahd.org/wp-content/uploads/2024/10/HydraHD-icon2.webp

### Prerequisites

- [ ] It is a paid service
- [ ] It is region restricted

### Description

Just a basic presence showing what show/movie you are watching along side what episode/timestamp you are at. 
Tried doing it myself but am smol brained. No rush and thank you in advance! :D

**Comment by DiscordAnaxes** [comment]
I'll give it a go 👍.

**Comment by DiscordAnaxes** [comment]
Hi @spoon-codes, I was just wondering does the website work anymore. I can't seem to access the website from any of the domains listed.

**Comment by Slowlife01** [comment]
> Hi @spoon-codes, I was just wondering does the website work anymore. I can't seem to access the website from any of the domains listed.

Yeah still works

**Comment by spoon-codes** [comment]
@DiscordAnaxes Yes the site still works. It seems when the thread name was changed, they accidentally edited the link also. I'll go ahead and change that back. Thank you in advance <3

**Comment by DiscordAnaxes** [comment]
> hydrahd.me

Thanks for the confirmation. I seem to have been region locked (currently overseas) for some odd reason.

**Comment by DiscordAnaxes** [comment]
> @DiscordAnaxes Yes the site still works. It seems when the thread name was changed, they accidentally edited the link also. I'll go ahead and change that back. Thank you in advance <3

Everything works with the activity, just can't get live sports to work - keep on receiving a network error on the streams. Just wanted to confirm if they work normally for everyone else, as I am on a VPN.

**Comment by spoon-codes** [comment]
@DiscordAnaxes I didn't even know the site supported live sports to be honest. After checking on my end, it seems 99% of the servers for them don't work. I'm also running into constant network errors. 

I'm not worried about the Live Sports part personally, but if you want to keep fighting it and trying to get it to work, more power to you
